### DISCUSSION TITLE
Show products before they go on sale

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretix/pretix#3929] [Q&A] [answered]
Hello,
I can't find the settings, so the question is, is it possible that products can already be displayed, even if they are not yet available?

At the moment they just "disappear" in the overview when the availability is set to the future.

**Comment by navlisilvan** [comment]
At the moment i use the workaround, that i set the contingent for this product to zero, write the date of availability in the text of the category and then have to manually increase the contingent on that day.

**Comment by pc-coholic** [comment, marked as answer]
Generally speaking, there are two options:

1. You can not set any availability dates on the products and set a "start of presale"-date on the shop.

2. With the newest version of pretix, you can set per-product availability and choose using the eye/info-button if the item should be hidden oder shown even when outside its availability.

![Screenshot_20240219_202003_com.android.chrome.jpg](https://github.com/pretix/pretix/assets/654548/70d1ff4e-e0cf-40d1-accf-e869e89e803e)

**Reply by pc-coholic** [reply]
Correction: this feature is on the master branch and will be officially available in the next release later this month

**Reply by navlisilvan** [reply]
2. is what i would like to use.
We have installed pretix 2024.1.0 but i cannot see this buttons

**Reply by navlisilvan** [reply]
ah, thanks!
